agent.go

```diff
@@ -892,33 +901,33 @@ func (p *pod) buildProcess(hyperProcess hyper.Process) (*process, error) {
 	return p.buildProcessWithoutTerminal(proc)
 }
 
-func (p *pod) runCmd(cmd hyper.HyperCmd, data []byte) error {
+func (p *pod) runCmd(cmd hyper.HyperCmd, data []byte) ([]byte, error) {
 	cb, exist := callbackList[cmd]
 	if exist == false {
```

Comment:
This check should not return an error anymore. Indeed, relying on what I have mentioned above, if we add this new Hyper command to a new map called `callbackWithReplyList`, we should not fail but instead, we should try this other list. If we cannot find for both lists, then we can really fail this time.
This would allow not to force all the other commands to return nil all the time because they have no byte to send back.